packages/editor/src/lib/Editor.svelte

```diff
@@ -52,14 +52,25 @@
 	}}
 />
 
+<!-- svelte-ignore a11y_no_static_element_interactions -->
 <div
 	class="container"
 	bind:this={container}
-	onfocusin={() => {
+	onpointerdown={() => {
+		workspace.enable_tab_indent();
+	}}
+	onkeydown={(e) => {
+		if (e.key !== 'Tab') {
+			workspace.enable_tab_indent();
```

Comment:
how do you get out though once you're in? Should we listen to the Escape key and disable tab indent then?

Reply:
nevermind, this seems to work already